home.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:io';
+import 'Login.dart';
+import 'Sign-up.dart';
+
+void main() {
+  home main = new home();
+}
+
+class home {

```

Comment:
we better use naming conventions here
```suggestion
class Home {

```